INSTALL.md

```diff
@@ -45,6 +45,13 @@
 
         tlmgr install collection-fontsrecommended
 
+## Chromeos
```

Comment:
Thanks!  Minor stylistic requests:

- blank line after the header
- hard-wrap lines to stay within 80 columns
- Shouldn't it be 'Chrome OS' instead of 'Chromeos'?

Reply:
Maybe also link directly to `http://skycocker.github.io/chromebrew/`